tests/unit/Parsers/DdCoordinateParserTest.php

```diff
@@ -18,6 +19,22 @@
 class DdCoordinateParserTest extends StringValueParserTest {
 
 	/**
+	 * @deprecated since 0.3, just use getInstance.
+	 */
+	protected function getParserClass() {
+		return 'DataValues\Geo\Parsers\DdCoordinateParser';
+	}
```

Comment:
What is the reason for deprecating this rather than removing it? Is something still using it?